src/linear_algebra/matrix/hermitian.lean

```diff
@@ -200,14 +200,11 @@ funext h.coe_re_apply_self
 
 /-- A matrix is hermitian iff the corresponding linear map is self adjoint. -/
 lemma is_hermitian_iff_is_symmetric [fintype n] [decidable_eq n] {A : matrix n n α} :
-  is_hermitian A ↔ linear_map.is_symmetric
-    ((pi_Lp.linear_equiv 2 α (λ _ : n, α)).symm.conj A.to_lin' : module.End α (pi_Lp 2 _)) :=
+  is_hermitian A ↔ A.to_euclidean_lin.is_symmetric :=
 begin
```

Comment:
This spelling (suggested in https://github.com/leanprover-community/mathlib/pull/14231#discussion_r881800195) was already type-correct, but has been folded into `to_euclidean_lin`.